std/format/internal/write.d

```diff
@@ -28,11 +28,15 @@ package(std.format):
     `bool`s are formatted as `"true"` or `"false"` with `%s` and as `1` or
     `0` with integral-specific format specs.
  */
-void formatValueImpl(Writer, T, Char)(auto ref Writer w, T obj, scope const ref FormatSpec!Char f)
+void formatValueImpl(Writer, T, Char)(auto ref Writer w, const(T) obj, scope const ref FormatSpec!Char f) // const(T) reduces number of instances
```

Comment:
Ok to remove the comment if needed.

Reply:
I think we should remove them. As I mentioned before, having the argument be `const` makes sense from an interface point of view. `format` should not modify the value that is being inserted.

Reply:
Ok, I'll remove them.